tests/genome/test_clust.py

```
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+"""
+* @Date: 2025-01-25 14:23:55
+* @LastEditors: hwrn hwrn.aou@sjtu.edu.cn
+* @LastEditTime: 2025-04-22 11:36:51
+* @FilePath: /genome/tests/genome/test_clust.py
+* @Description:
+"""
+# """
+
+from tests import Path, temp_output, test_files, test_temp
```

Ruff (0.11.9)

`tests.test_files` imported but unused

Remove unused import

(F401)